nyc_taxis/challenges/default.json

```diff
@@ -737,6 +749,103 @@
           },
           "tags": ["setup"]
         },
+        {
+          "operation": "index-enrich-sources",
+          "tags": ["setup"]
+        },
+        {
+          "name": "check-enrich-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('yellow')}}",
+              "wait_for_no_relocating_shards": "true"
+            },
+            "retry-until-success": true
+          },
+          "tags": ["setup"]
+        },
```

Comment:
Thank you for changing the number of replicas. We can now switch to `green` here.

Reply:
Trying to address via https://github.com/elastic/rally-tracks/pull/469/commits/0bc6526736ac770997ca6e22e53a14e8718c0701.